src/SignUp.tsx

```diff
@@ -0,0 +1,152 @@
+// import axios from 'axios';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router';
+
+const SignUp = () => {
+	const navigate = useNavigate();
+	const [errors, setErrors] = useState('');
+	const [values, setValues] = useState({
+		firstName: '',
+		lastName: '',
+		email: '',
+		password: '',
+		confirmPassword: '',
+	});
+
+	const handleChange = (e: { target: { name: any; value: any } }) => {
+		const { name, value } = e.target;
+		setValues({
+			...values,
+			[name]: value,
+		});
+	};
+
+	const handleValidation = () => {
+		if (
+			values.firstName === '' ||
+			values.lastName === '' ||
+			values.email === '' ||
+			values.password === '' ||
+			values.confirmPassword === ''
+		) {
+			setErrors('some field are incomplete');
+		} else if (
+			values.confirmPassword.length < 3 &&
+			values.password.length < 3
+		) {
+			setErrors('password must be more then 3 characters');
+		} else if (values.password !== values.confirmPassword) {
+			setErrors('Password and Confirm Password does NOT match');
+		} else if (values.email.includes('@') === false) {
+			setErrors('Email invalid');
+		} else {
+			// axios
```

Comment:
Are these comments necessary?

Reply:
This is commented out because the backend for now is not running, If is not commented out every time the use press the submit button is going to consoleLog a 400 in developers tool. So i though it would be better i comment them out for now.

Reply:
Understood, thanks